nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -193,7 +193,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             cfg.joint.num_classes = len(vocabulary)
             cfg.joint.vocabulary = ListConfig(list(vocabulary))
             cfg.joint.jointnet.encoder_hidden = cfg.model_defaults.enc_hidden
-            cfg.joint.jointnet.pred_hidden = cfg.model_defaults.pred_hidden
+            cfg.joint.jointnet.pred_hidden = cfg.model_defaults.get('proj_size', cfg.model_defaults.pred_hidden)
```

Comment:
pred_hidden is necessary for all rnnt configs, why not just use that instead of yet another variable called proj_size?